api/net/Areas/Editor/Models/Storage/ItemModel.cs

```diff
@@ -32,6 +33,12 @@ public class ItemModel
     /// get/set - The media-type of the file.
     /// </summary>
     public string? MimeType { get; set; }
+
+    /// <summary>
+    /// get/set - The media-type of the file.
+    /// </summary>
+    public string? modified { get; set; }
```

Comment:
Pascal case

Reply:
In C# should I use Pascal case for every variable/attribute/method? Is there a style guide that you use?

Reply:
Fixed.